src/main/java/org/wildfly/security/auth/client/ElytronXmlParser.java

```diff
@@ -1095,7 +1096,7 @@ private static int setBit(int var, int bit) {
                         break;
                     }
                     case "local-kerberos": {
-                        if ( ! xmlVersion.isAtLeast(Version.VERSION_1_1)) {
+                        if ( ! xmlVersion.isAtLeast(Version.VERSION_1_1) || xmlVersion.isAtLeast(Version.VERSION_1_2)) {
```

Comment:
I am not sure if we need to bump the schema just for the deprecation but we should not need the OR here as isAtLeast should detect 1.2 is 'at least' 1.1

Reply:
Ok I see, at this stage I don't think we need to add the 1_2 schema and we will handle the removal later.